eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -4,6 +4,9 @@
     "http://www.puppycrawl.com/dtds/suppressions_1_0.dtd">
 
 <suppressions>
+  <!-- Don't scan template module -->
+  <suppress checks=".*" files="com[\\/]microsoft[\\/]azure[\\/]template[\\/]"/>
```

Comment:
Is this still needed? Also does it hurt to scan the template?

Reply:
We can remove this for now, until it becomes a problem...